packages/dashboard/src/components/composed/Analytics.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  Stack,
+  Anchor,
+  Space,
+  createStyles,
+  Accordion,
+  Switch,
+  Grid
+} from "@mantine/core";
+import { ExternalLink } from "react-feather";
+import { useDash } from "src/hooks";
+import { useState, useEffect } from "react";
+import { analyticsNotifications } from "src/utils/notifications";
+import { showNotification } from "@mantine/notifications";
+
+const useStyles = createStyles((_theme, _params, _getRef) => ({
+  maxSize: {
+    width: "100%",
+    height: "100%"
+  },
+  docsAnchor: {
+    display: "inline-flex",
+    alignItems: "center"
+  }
+}));
+
+function Analytics(): JSX.Element {
+  const {
+    state: { analyticsConfig },
+    operations: { updateAnalyticsConfig }
+  } = useDash()!;
+  const analyticsValue = analyticsConfig.enableAnalytics ? true : false;
+
+  const [checked, setChecked] = useState(analyticsValue);
+
+  const { classes } = useStyles();
+
+  useEffect(() => {
+    setChecked(analyticsConfig.enableAnalytics ? true : false);
+  }, [analyticsConfig]);
+  return (
+    <Stack>
+      <Accordion radius="md" defaultValue="Telemetry">
+        <Accordion.Item value="telemetry">
+          <Accordion.Control>Telemetry</Accordion.Control>
+
+          <Grid grow>
+            <Grid.Col span={8}>
+              <Accordion.Panel>
+                You can help us improve Truffle by allowing us to track how you
+                use our tools. Would you like to share anonymous telemetry? View
+                the Truffle Analytics policy&nbsp;
+                <Anchor
+                  href="https://trufflesuite.com/analytics/"
+                  target="_blank"
+                  color="truffle-beige"
+                  inherit
+                  className={classes.docsAnchor}
+                >
+                  here
+                  <Space w={3} />
+                  <ExternalLink size={12} />
+                </Anchor>
```

Comment:
We shouldn't use words like "here" to link places when you can add a link to the thing itself:

```suggestion
                the 
                <Anchor
                  href="https://trufflesuite.com/analytics/"
                  target="_blank"
                  color="truffle-beige"
                  inherit
                  className={classes.docsAnchor}
                >
                  Truffle Analytics policy
                  <Space w={3} />
                  <ExternalLink size={12} />
                </Anchor>
```

note: I don't know what ExternalLink and Space are